### DISCUSSION TITLE
no acrylic blur effect with ExperimentalAcrylicBorder

### DISCUSSION THREAD

**Discussion by FanIT** [AvaloniaUI/Avalonia#5604] [Q&A] [answered]
Hello. I'm trying to use "ExperimentalAcrylicBorder" to get an acrylic blur effect in my app like Windows 10. But the effect appears for one second after starting the app only, then black background simply.
Also i runned your test app XamlControlGallery and I had similar behavior.
I would like to know. What versions of Windows 10 does it work?
My os Windows 10 1809

```
<Window xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        x:Class="SysMonAvalonia.Widgets.DiscWidget"
        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
        Title="DiscWidget" Width="360" ShowInTaskbar="False" CanResize="False" 
        SystemDecorations="None" WindowStartupLocation="CenterScreen"
        TransparencyLevelHint="AcrylicBlur" Background="{x:Null}" SizeToContent="Height" Opened="DiscWidget_Opened">
  <Grid>  
    <ExperimentalAcrylicBorder>
      <ExperimentalAcrylicBorder.Material>
        <ExperimentalAcrylicMaterial BackgroundSource="Digger" TintOpacity="0,5" TintColor="Black" MaterialOpacity="0.85"/>
      </ExperimentalAcrylicBorder.Material>
    </ExperimentalAcrylicBorder>
    <StackPanel x:Name="DiscPanel" Orientation="Vertical" />
  </Grid>
  </Window>
```
My test application.
![mytestapp](https://user-images.githubusercontent.com/20283004/109826154-90e12e00-7c4b-11eb-90a8-b65a70fc2375.png)

XamlControlGallery.
![Screen](https://user-images.githubusercontent.com/20283004/109823998-79a14100-7c49-11eb-8be1-9a522e67df07.png)

**Comment by danwalmsley** [comment, marked as answer]
@FanIT minimum supported windows 10 build number is:

`bool canUseAcrylic = Win32Platform.WindowsVersion.Major > 10 || Win32Platform.WindowsVersion.Build >= 19628;`

https://github.com/AvaloniaUI/Avalonia/blob/master/src/Windows/Avalonia.Win32/WindowImpl.cs#L366

 I think it may be that you need a newer build for acrylic to work on your machine, its falling back to FallbackColor.

Even though UWP are showing acrylic blur, they are doing this via an internal api that Avalonia doesnt have access to, which is why we require even newer build.

**Comment by FanIT** [comment]
@danwalmsley I had suspected that it wouldn't work for old versions Windows 10.
But i found a similar solution. 
If usen't `ExperimentalAcrylicBorder` and to add `Rectangle` with `Opacity="0.5"`

```
<Window xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        x:Class="SysMonAvalonia.Widgets.DiscWidget"
        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
        Title="DiscWidget" Width="360" ShowInTaskbar="False" CanResize="False" 
        SystemDecorations="None" WindowStartupLocation="CenterScreen"
        TransparencyLevelHint="AcrylicBlur" Background="{x:Null}" SizeToContent="Height" Opened="DiscWidget_Opened">
  <Grid>
    <Rectangle Fill="Black" Stroke="Black" Opacity="0.5" />
    <StackPanel x:Name="DiscPanel" Orientation="Vertical" />
  </Grid>
  </Window>
```
![mytestapp2](https://user-images.githubusercontent.com/20283004/109931159-8c635680-7cd9-11eb-86cd-036a84928647.png)

Thanks for the clarification.

**Comment by FanIT** [comment]
@danwalmsley I have one more question. Why hiding a icon on taskbar doesn't work?
I set ShowInTaskbar="False", but the icon appears still.

**Reply by Gillibald** [reply]
You can't hide the Icon for a root level window

**Reply by FanIT** [reply]
@Gillibald yes.
